README.md

```diff
@@ -74,3 +74,67 @@ is as follows
 ## Merging into Main
 Once a PR is merged into the main branch the snapshot suffix will be deleted from the version.
 The assigned version number will be used to create a version release in artifactory and a Github tag for backup.
+
+
+## Using Filters
+
+### Overview
+JpaRest supports the use of filtering data during retrieval for certain get endpoints. These filters should be constructed in accordance with Spring Expression Language (SpEL) as they will then be converted into predicates used to query the database.
```

Comment:
I like how you format GET below, can we make it consistent?
```suggestion
JpaRest supports the use of filtering data during retrieval for certain `GET` endpoints. These filters should be constructed in accordance with Spring Expression Language (SpEL) as they will then be converted into predicates used to query the database.
```

Reply:
ah yes missed that one. updated.